account_statement_import_camt54/README.rst

```diff
@@ -0,0 +1,90 @@
+==========================
```

Comment:
this file needs to be regenerated to have the good module name and Odoo main version

Reply:
regeneration will be done by bot, you need to take care only on the files in readme folder